package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "lint": "aegir-lint",
-    "test": "aegir-test --env node",
+    "test": "aegir-test --env node --timeout 20000",
```

Comment:
Why do we need this now? Have timeouts increased with this changes? If so, we should have individual timeouts.

Reply:
Yes a lot of tests were skipped.  In particular the `many (3 * 400)` test takes a long time on my top-end laptop when sharding.